UnrealPlugin/README.md

```diff
@@ -47,4 +123,52 @@ public class <projectname>ServerTarget : TargetRules
 	}
 }
 
-```
\ No newline at end of file
+```
+
+Update DefaultGame.ini to show the following:
+ [/Script/UnrealEd.ProjectPackagingSettings]
+IncludeAppLocalPrerequesites=True
+
+If the category already exists in your DefaultGame.ini, then just add the second line to it. This ensures that all app local dependencies ship with the game as well.
+
+Update DefaultEngine.ini file and add this:
+[/Script/EngineSettings.GameMapsSettings]
+GameInstanceClass=/Script/[game name].MyGameInstance"
+
+Update [game name].Build.cs file to add "PlayfabGSDK" into the PublicDependencyModuleNames.AddRange(); list as follows:
+
+```csharp
+		PublicDependencyModuleNames.AddRange(new string[] { "Core", "CoreUObject", "Engine", "InputCore", "HeadMountedDisplay", "PlayfabGSDK"});
+
+		PrivateDependencyModuleNames.AddRange(new string[] { });
+```
+
+Then select the option to build from development editor mode and build the project.
+
+![image depicting Visual Studio with the option to build in Development Editor Mode](Resources/UnrealTutorialImages/DevelopmentEditor.png)
+
+Then select the option to build from development server mode and build the project again.
+
+Once the build succeeds, close Visual Studio.
+
+## Packaging the game server
+
+Right click on the .uproject file in your file editor and select "Generate Visual Studio Project Files". 
+
+Then open the .uproject file.
+
+In the top left of the Unreal editor menu, click on file->package->target configuration ->[game name] server and then go to file -> package -> windows x64. 
+You can now use this packaged version of your game server to [test with LocalMultiplayerAgent](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/locally-debugging-game-servers-and-integration-with-playfab) 
+or to use it directly with [Playfab VMAgent by using the wrappingGSDK](https://github.com/PlayFab/MpsSamples/blob/master/wrappingGsdk/README.md).
```

Comment:
isnt there wrapper for wrapping a projcet that does not integrate with PlayfabGSDK?